glossary.asciidoc

```diff
@@ -279,18 +279,18 @@ Payment Channel::
     There are currently three methods known to construct a fully duplex bidirectional payment channel.
     Christian Decker proposed a method in his PhD thesis based on invalidation trees.
     In the Lightning Network whitepaper, Joseph Poon and Tadge Dryja describe the Revocable Sequence Maturity Contract based method that is currently being implemented on the Lightning Network.
-Recently Christian Decker et al. came up with the Eltoo mechanism which would require a Bitcoin softfork.
+Recently Christian Decker et al. Came up with the Eltoo mechanism which would require a Bitcoin softfork.
```

Comment:
I think lower case "came" is correct, this isn't a new sentence.   maybe it would be better to use the word "proposed" instead of "came up with".  Thx Bitcoina.

Reply:
Thanks for the advice.